src/lib/components/Card/Card.tsx

```diff
@@ -57,14 +64,37 @@ export const Card: FC<CardProps> = ({
       )}
       {...theirProps}
     >
-      {imgSrc && (
-        <img
-          alt={imgAlt ?? ''}
-          src={imgSrc}
-          className={classNames(theme.img.base, theme.img.horizontal[horizontal ? 'on' : 'off'])}
-        />
-      )}
+      <Image {...props} />
       <div className={theme.root.children}>{children}</div>
     </Component>
   );
 };
+
+const Image: FC<CardProps> = ({ theme: customTheme = {}, ...props }) => {
+  const theme = mergeDeep(useTheme().theme.card, customTheme);
+  if (props.renderImage) {
+    return props.renderImage();
+  }
+  if (props.imgSrc) {
+    return (
+      <img
+        data-testid="flowbite-card-image"
```

Comment:
Usage of testId is a little uncool, but I could find a better way to check that "this component is not rendered" from the tests.